refinery/analysis_manager/tasks.py

```diff
@@ -486,121 +419,9 @@ def _run_tool_based_galaxy_workflow(analysis_uuid):
         return
```

Comment:
I've just continued using these returns as they we're being used beforehand in this area of the codebase. If you are confident that I can remove them then I can.

Reply:
They were necessary previously because they provided exit points in the middle of the function. Now they are all at the end of their respective functions, so they don't have any effect.